tests/wpt/meta/page-visibility/iframe-session-history.html.ini

```diff
@@ -0,0 +1,3 @@
+[iframe-session-history.html]
+  [pagehide should be called before visibilitychange, and visibilityState should be set to hidden at the right time]
```

Comment:
I think this one probably goes to the core of the issue you opened, would be good to link to it in the issue .